docs/data-destinations/grafana-cloud-setup.md

```diff
@@ -59,3 +59,17 @@ It depends on your Grafana Cloud version the information that you need to share
 4. Share the instance ID and zone with an Embrace onboarding specialist.
 
 <img src={require('@site/static/images/grafana_cloud_details.png').default} alt="Image showing Grafana Cloud fields needed" />
+
+## Selecting the Correct Data Source for Embrace Metrics
+
+Embrace provides two main options for sending metrics to your Grafana Cloud instance: using the **Embrace Metrics API** (pull method) or setting up a **Data Destination** (push method).
```

Comment:
For Embrace Metrics API, we are not sending metrics to Grafana Cloud, you can consume them from Grafana Cloud.

Reply:
Updated